### PR TITLE
Fixed 'it' language translation

### PR DESCRIPTION
**Pull request by Matt75**
- fixed existing translation from 'fr' to 'it' language.
- added missing translations retrieved from 'fr.php'.

Now 'fr.php' and 'it.php' have the same variables.

Cherry pick of @flavioski pull request

### CODE REVIEW

File: translations/it.php [hunk 1]
```diff
@@ -334,9 +343,11 @@
...
 $_MODULE['<{ps_checkout}prestashop>translations_19146f30e246bba7235f1f052711fe74'] = 'Qual è la differenza tra i campi carta di credito integrati e i campi della carta di credito PayPal?';
 $_MODULE['<{ps_checkout}prestashop>translations_ff8313e7d0a38ececa23e28fd77421de'] = 'Quando usare i campi della carta di credito PayPal?';
 $_MODULE['<{ps_checkout}prestashop>translations_4cb668c4f89af857decf3d6ce310cd77'] = 'I campi carta di credito integrati offrono la migliore esperienza di pagamento possibile su PrestaShop. Ben integrati nel processo di pagamento, privi di marca, con il minor numero di campi possibile e le tariffe più basse (vedi i campi su';
+$_MODULE['<{ps_checkout}prestashop>translations_3a8e02212752fecc2b5eb72111cdc40d'] = '): ti consigliamo di dare la preferenza a loro. Il tuo account PayPal deve essere stato completamente approvato per trarne vantaggio. Puoi vedere lo stato di approvazione del tuo account PayPal nel file';
```
Comment (seiwan) [inline translations/it.php:350]: There is a blank space for this translation on en.php and fr.php files.
Also, it doesn't exists on other languages, maybe it's the same thing for some other translations. @Darmona

Comment (flavioski) [reply translations/it.php:350]: Hi, yes you are right. :)
But I think there is an error about the fr.php language.
I mean, I think, this sentence is the second part of the first part indicated on previously line (349) so, in that line there is no white space after the opening brakets. This is the reason there is no white space here.

Comment (seiwan) [reply translations/it.php:350]: Hello !
In fact, I think there is no errors on english and french files. This is a syntax convention. By example, on your comment you added blank spaces before and after opening and closing brackets: "line (349) so". :)

**Review by seiwan — DISMISSED** [review]

File: translations/it.php (hunk 1 above)
Comment (flavioski) [reply translations/it.php]: Sorry now I understand. My apologies.
Thank you :)

**Review by seiwan — DISMISSED** [review]

**Pull request merged**